src/components/Modals/UploadButtonGroup.jsx

```diff
@@ -0,0 +1,50 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import PhotoCameraIcon from '@mui/icons-material/PhotoCamera';
+import SearchIcon from '@mui/icons-material/Search';
+
+const UploadButtonGroup = ({ file, setFile }) => (
+  <Box sx={{ display: 'flex', padding: '8px 8px 0', boxSizing: 'border-box' }}>
+    <Button
+      variant={file ? 'outlined' : 'contained'}
+      component="label"
+      color="primary"
+      id="upload-doctype"
+      name="uploadDoctype"
+      onChange={(e) => setFile(e.target.files[0])}
```

Comment:
Can you explain this line? I'm missing something here... can we upload multiple files?  Trying to understand the `e.target.files[0]`

Reply:
`e.target.files` is an object that could store multiple files objects (you would have to turn on the multiple attribute in `<input>`), `e.target.files[0]` is simply the file object stored at the key 0. For individual uploads, we could access the file we want with `e.target.files[0]`.

![Screen Shot 2023-08-01 at 15 14 17](https://github.com/codeforpdx/PASS/assets/14917816/0bc5fb75-8676-4b36-ad16-f244f62da973)

Reply:
Ok. Wouldn’t we always want to upload every file that the user selects to upload?

Reply:
We could, but we don't have the `multiple` attribute turned on at the moment (see Issue #77). And without the `multiple` attribute, users cannot select more than one file from the file explorer.

Clip without `multiple` attribute for `<input>`

https://github.com/codeforpdx/PASS/assets/14917816/a71a7314-58c5-44eb-af7e-a0041dc26f2c

Clip with `multiple` attribute for `<input>`

https://github.com/codeforpdx/PASS/assets/14917816/ff6211d5-f952-408d-a288-4d554d162587

On top of that, we haven't really discussed a way to support bulk upload yet with PASS and Solid.